packages/next/pages/_document.tsx

```diff
@@ -498,12 +510,12 @@ export class Head extends Component<
                 href={canonicalBase + getAmpPath(ampPath, dangerousAsPath)}
               />
             )}
-            {process.env.__NEXT_OPTIMIZE_FONTS
-              ? this.makeStylesheetInert(this.getCssLinks(files))
-              : this.getCssLinks(files)}
-            <noscript data-n-css />
+            {!process.env.__NEXT_OPTIMIZE_CSS && this.getCssLinks(files)}
```

Comment:
these repetitive calls to `this.getCssLinks` could use some documentation around the ordering